intelmq/tests/bots/parsers/shadowserver/test_scan_synfulknock.py

```diff
@@ -0,0 +1,114 @@
+# SPDX-FileCopyrightText: 2019 Guillermo Rodriguez
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+# -*- coding: utf-8 -*-
+
+import os
+import unittest
+
+import intelmq.lib.test as test
+import intelmq.lib.utils as utils
+from intelmq.bots.parsers.shadowserver.parser import ShadowserverParserBot
+
+with open(os.path.join(os.path.dirname(__file__),
+                       'testdata/scan_synfulknock.csv')) as handle:
+    EXAMPLE_FILE = handle.read()
+EXAMPLE_LINES = EXAMPLE_FILE.splitlines()
+
+EXAMPLE_REPORT = {'feed.name': 'SYNful Knock',
+                  "raw": utils.base64_encode(EXAMPLE_FILE),
+                  "__type": "Report",
+                  "time.observation": "2022-01-07T00:00:00+00:00",
+                  "extra.file_name": "2022-01-07-scan_synfulknock-test.csv",
+                  }
+EVENTS = [
+{
+   '__type' : 'Event',
+   'classification.identifier' : 'scan-synfulknock',
+   'classification.taxonomy' : 'vulnerable',
+   'classification.type' : 'vulnerable-system',
+   'extra.ack_number' : '791102',
+   'extra.raw_packet' : '3cfdfec601e4700f6a9a2000080045000034c3780000f706789442099555b869f7ee0050b20800000000000c123e8012200002aa0000020405b40101040201030305',
+   'extra.tag' : 'synfulknock',
+   'extra.tcp_flags' : '4608',
+   'extra.window_size' : '8192',
```

Comment:
could this be int?

Reply:
Keeping tcp_flags as a string as the implementation could change.

Fixed:
```
-    'extra.window_size' : '8192',
+    'extra.window_size' : 8192,